interview-questions.md

```diff
@@ -8,36 +8,63 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: The foreign key would be on the student model as it would be calling the primary id from the cohort model as the relational link. If cohort has_many students, that would mean Student belongs_to Cohort, making Cohort the parent and Student the child of Cohort. We can fix this mistake by using migrations to modify our schema. I have personally already had to do this, changing the datatype of a certain column or adding a new one.
 
-Researched answer:
+Researched answer: After creating a new migration with the rails generate migration command, we can define, within the migration created, a change method to update what we wish. If we neglected to add a column, we can simply use an add_column statement and add our student_id to act as the foreign key, as well as the associated data type for the column.
```

Comment:
The foreign key would be referencing the primary key from the parent so that would be `cohort_id:integer`